endpoint.cpp

```diff
@@ -545,6 +545,7 @@ TcpEndpoint::TcpEndpoint()
 
 TcpEndpoint::~TcpEndpoint()
 {
+    close();
```

Comment:
I guess that `fd` is closed by `Pollable` destructor - but if this is for the logging, ok.